operators/meetings/041-20230713.md

```diff
@@ -0,0 +1,78 @@
+* Christian Bourjau (QuantCo)
+* Christophe Gabreau (Airbus)
+* Donald Tolley (GResearch)
+* Ganesan Ramalingam (Microsoft)
+* George (Intel)
+* Jacky Chen (Microsoft)
+* Joaquin Anton (Nvidia)
+* Justin Chu (Microsoft)
+* Liqun Fu (Microsoft)
+* Marie-Charlotte Teulieres (Airbus)
+* Michal Karzynski (Intel)
+* Xavier Dupré (Microsoft)
+* Yuan Yao (Nvidia)
+
+## Notes
+
+### Airbus collaboration: Machine Learning Model Description (MLMD) Requirements for ONNX Evaluation
+
+Airbus is working on integrating AI systems for safety-critical systems, specifically for embedding in airplanes.
+This project necessitates a certification process, adhering to the 
+[ARP6983](https://www.sae.org/standards/content/arp6983/) standard. 
+
+A vital component of this project is the Machine Learning Model Description (MLMD) which 
+defines the interface between software and hardware. Airbus has identified ONNX as a potential platform
+to support the MLMD, citing its ability to provide explicit and comprehensive semantics for all operations in the model.
+
+The MLMD's objective is to support the exact replication of a model from its design phase
+to the final implementation. This functionality is particularly important 
+for Machine Learning Constituents (MLC) - subsystems containing the models.
+
+
+### Criteria for ONNX
+
+The key selection criteria for ONNX, as outlined by Marie-Charlotte Teulieres of Airbus, includes interoperability, 
+and the explicit description of semantics involving computational graphs and all operators. 
+Airbus pointed out the necessity to complete the description of all mathematical formulas for all operations. 
+
+Airbus offered positive feedback regarding ONNX, albeit with three suggestions for enhancement.
+
+1. **Explicit Unambiguous Description of Operator Semantics:** 
+Airbus suggests incorporating mathematical formulas to provide explicit, unambiguous descriptions
+of operator semantics. As a starting point, creating a subset of operators fulfilling all constraints,
+such as considering overflow for data types, was suggested. Alexandre and Michal proposed it would be valuable 
+for the formulas to be specified in a language that is not only human-readable but also machine-readable.
+This would allow the formulas to be used for testing and verification purposes.
+While Python implementations are useful, they were deemed insufficient.
+
+2. **Human-Readable Format:** 
+Airbus pointed out that a human-readable text representation of the models is a requirement. 
+The textual representation of protocol buffers, which is currently available is cluncky and difficult to read.
+Rama brought up the ONNX markup format, which is easier and may be a good candidate for this purpose. 
+Further work on this format is needed to make it more complete and usable.
+It was suggested that there might be a future need to transition away from Protobuf.
+
+Airbus pointed out that a human-readable text representation of the models is a requirement. 
```

Comment:
This seems like a duplicate of previous paragraph, though with minor wording changes. So, I am not sure which one you want to retain.

Reply:
I was re-editing this paragraph, and forgot to remove the original. Now removed.